<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1235</width>
    <height>105</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">Сапёр</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="lcdMineCount">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_4">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item row="0" column="1">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QPushButton" name="controlButton">
            <property name="text">
             <string/>
            </property>
            <property name="iconSize">
             <size>
              <width>48</width>
              <height>48</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="0" column="2">
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="lcdTimer">
            <property name="text">
             <string>00:00</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QGridLayout" name="buttonGrid">
        <property name="spacing">
         <number>0</number>
        </property>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1235</width>
     <height>30</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string notr="true">Файл</string>
    </property>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuGame">
    <property name="title">
     <string notr="true">Игра</string>
    </property>
    <widget class="QMenu" name="customGameMenu">
     <property name="title">
      <string notr="true">Пользовательская</string>
     </property>
     <addaction name="actionNewCustomGame"/>
    </widget>
    <addaction name="actionBigGame"/>
    <addaction name="actionMediumGame"/>
    <addaction name="actionSmallGame"/>
    <addaction name="separator"/>
    <addaction name="customGameMenu"/>
    <addaction name="separator"/>
    <addaction name="actionResults"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string notr="true">Справка</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuGame"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="actionExit">
   <property name="text">
    <string notr="true">Выход</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string notr="true">О программе...</string>
   </property>
  </action>
  <action name="actionBigGame">
   <property name="text">
    <string notr="true">big</string>
   </property>
  </action>
  <action name="actionMediumGame">
   <property name="text">
    <string notr="true">medium</string>
   </property>
  </action>
  <action name="actionSmallGame">
   <property name="text">
    <string notr="true">small</string>
   </property>
  </action>
  <action name="actionResults">
   <property name="text">
    <string notr="true">Результаты</string>
   </property>
  </action>
  <action name="actionNewCustomGame">
   <property name="text">
    <string notr="true">Новая...</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
